Sources/Private/Utility/Primitives/VectorsExtensions.swift

```diff
@@ -339,3 +339,26 @@ extension CATransform3D {
     CATransform3DConcat(CATransform3D.makeSkew(skew: skew, skewAxis: skewAxis), self)
   }
 }
+
+// MARK: - CATransform3D + Equatable
+
+extension CATransform3D: Equatable {
```

Comment:
If someone else has this conformance will this definition conflict? Wondering if we could avoid this if so

Reply:
LayerProperty has an Equatable constraint right now. Good point about this conformance potentially causing conflicts. I'll look at making the Equatable constraint optional.

Reply:
https://github.com/airbnb/lottie-ios/pull/1917/commits/82ffdf284a4990928d98ff8ac799c34e5fcb20b0